tests/Configurator/DockerComposeConfiguratorTest.php

```diff
@@ -0,0 +1,325 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Tests\Configurator;
+
+use Composer\Composer;
+use Composer\IO\IOInterface;
+use Composer\Package\Package;
+use PHPUnit\Framework\TestCase;
+use Symfony\Flex\Configurator\DockerComposeConfigurator;
+use Symfony\Flex\Lock;
+use Symfony\Flex\Options;
+use Symfony\Flex\Recipe;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class DockerComposeConfiguratorTest extends TestCase
+{
+    const ORIGINAL_CONTENT = <<<'YAML'
+version: '3.4'
+
+services:
+  app:
+    build:
+      context: .
+      target: symfony_docker_php
+      args:
+        SYMFONY_VERSION: ${SYMFONY_VERSION:-}
+        STABILITY: ${STABILITY:-stable}
+    volumes:
+      # Comment out the next line in production
+      - ./:/srv/app:rw,cached
+      # If you develop on Linux, comment out the following volumes to just use bind-mounted project directory from host
```

Comment:
Can be simplified.

https://github.com/api-platform/api-platform/blob/6a2ff40552d7d7b008c698cd120eea7f01fb1247/docker-compose.yml#L22-L23

Something like `var/jwt` should not be a problem, as a more specific path would take precedence. (But we need to verify that `setfacl` works...)

Reply:
It's just a fixture file, its content doesn't matter.